bigflow/scaffold/scaffold_templates.py

```diff
@@ -0,0 +1,348 @@
+readme_template = '''# Run project
+
+### Requirements
+  1. [GCP SDK](https://cloud.google.com/sdk/docs).
+
+### Preparations
+  1. Log in using SDK (`gcloud auth application-default login`).
```

Comment:
commands should not be standalone not inlined, so :

Log in to GCP using Google Cloud SDK:

```
gcloud auth application-default login
```